.github/workflows/tf-account.yaml

```diff
@@ -50,6 +72,8 @@ jobs:
 
       - name: Terraform Init
         id: init
+        # We avoid proxying on init, otherwise the exclusion
```

Comment:
This should be configurable and probably enabled by default, no? If you're proxying it seems like you should be proxying all traffic by default.

Reply:
That was my first take, but if we only want to proxy traffic intended for the Observe control plane you end up having quite a long list in `NO_PROXY`.